tests/test_shacl.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from pyshacl import validate
+
+
+@pytest.mark.parametrize(
+    ("shacl", "example"),
+    [
+        ("Catalog", "example-catalog"),
+        ("Dataset", "example-dataset"),
+        ("Distribution", "example-distribution"),
+    ],
+)
+def test_core_shapes(shacl, example):
```

Comment:
The name of the functions is not what you do.... i think the name shoudl be test_graph_conforms_shacl or something like that..

Reply:
This is the name that shows up in the rest report. I think `test_graph_conforms_shacl` might not be super descriptive, but open to suggestions.